sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Renamed the model `Appearance` to `TextAppearance`.
 - Renamed the model `Style` to `TextStyle`.
 - Renamed the extensible enum `TextStyle` to `TextStyleName`.
+- Changed object type for property `Pages` under `RecognizeContentOptions` from `IEnumerable` to `IList`.
```

Comment:
Curious what is the return type for `RecognizedForm.pages` is it IEnumerable or IList?

Was checking this in Java too, we have supported List for both rather than Iterable (IEnumerable) equivalent.

Reply:
It is a `IReadOnlyList` because it is an output model